networking_ccloud/common/exceptions.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 SAP SE
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from neutron_lib import exceptions as n_exc
+
+
+class MultipleBindingHostsInBindingProfile(n_exc.NeutronException):
+    """An exception indicating multiple hosts found in binding profile"""
+    message = "Port %(port_id)s has multiple hosts in binding profile, refusing to work with it! Hosts were: %(hosts)s"
```

Comment:
Please change the `, refusing` to `. Refusing` or ` - refusing` or `; refusing`. Otherwise, the hosts would not like to work with the ports. My understanding is, that the driver doesn't like to work with the port.